cores/esp8266/FS.cpp

```diff
@@ -441,6 +441,13 @@ bool FS::rename(const String& pathFrom, const String& pathTo) {
     return rename(pathFrom.c_str(), pathTo.c_str());
 }
 
+time_t FS::creationTime() {
+    if (!_impl) {
+        return -1;
```

Comment:
Return 0 like File::getCreationTime please.

Reply:
Sure, just thought it might be a nice way to check for errors vs valid values. I'll change it to `return 0`.